src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleCRUDAnonTests.java

```diff
@@ -57,37 +55,34 @@ public void ArticleCRUDAnon_003_addDropdown() {
 
   @Test(dataProviderClass = ArticleDataProvider.class, dataProvider = "articleTitles",
       groups = {"ArticleCRUDAnon_004"})
-  @RelatedIssue(issueID = "MAIN-6278", comment = "test manually")
   public void ArticleCRUDAnon_004_differentTitles(String articleTitle) {
     String articleContent = PageContent.ARTICLE_TEXT;
     String randomArticleTitle = articleTitle + DateTime.now().getMillis();
     VisualEditModePageObject visualEditMode =
         new ArticlePageObject().navigateToArticleEditPage(wikiURL, randomArticleTitle);
     visualEditMode.addContent(articleContent);
-    ArticlePageObject article = visualEditMode.submitArticle();
+    ArticlePageObject article = visualEditMode.submitArticleAsAnon();
     article.verifyContent(articleContent);
     article.verifyArticleTitle(randomArticleTitle);
   }
 
   @Test(groups = {"ArticleCRUDAnon_005"})
-  @RelatedIssue(issueID = "MAIN-6278", comment = "test manually")
   public void ArticleCRUDAnon_005_editByURL() {
     String articleContent = PageContent.ARTICLE_TEXT;
     ArticlePageObject article = new ArticlePageObject().open("AnonEditByURL");
     VisualEditModePageObject visualEditMode = article.navigateToArticleEditPage();
     visualEditMode.addContent(articleContent);
-    visualEditMode.submitArticle();
+    visualEditMode.submitArticleAsAnon();
     article.verifyContent(articleContent);
   }
 
   @Test(groups = {"ArticleCRUDAnon_006"})
-  @RelatedIssue(issueID = "MAIN-6278", comment = "test manually")
   public void ArticleCRUDAnon_006_editDropdown() {
     String articleContent = PageContent.ARTICLE_TEXT;
     ArticlePageObject article = new ArticlePageObject().open("AnonEditDropdown");
     VisualEditModePageObject visualEditMode = article.editArticleInCKUsingDropdown();
     visualEditMode.addContent(articleContent);
-    visualEditMode.clickPublishButton();
+    visualEditMode.submitArticleAsAnon();
     article.verifyContent(articleContent);
   }
 }
```

Comment:
Add a test for notification not appearing for the second time an anon edits a page